extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -685,6 +686,16 @@ export class AmpAutocomplete extends AMP.BaseElement {
     const index = keyUpWhenNoneActive ? delta : this.activeIndex_ + delta;
     const activeIndex = mod(index, this.container_.children.length);
     const newActiveElement = this.container_.children[activeIndex];
+
+    // Mark the next element as active if the current is 'disabled'
+    if (newActiveElement.hasAttribute('disabled')) {
+      if (this.container_.children.length === 1) {
+        return;
+      }
+      const newDelta = delta < 0 ? delta - 1 : delta + 1;
+      return this.updateActiveItem_(newDelta);
```

Comment:
If the full list of elements is disabled, could this cause an infinite recursion?

Reply:
Haha, yes! Good thing it is gone now. 😄